openfisca_tasks/install.mk

```diff
@@ -1,13 +1,19 @@
+## Install overall dependencies
+install-deps:
```

Comment:
If I understand correctly, `openfisca-country-template` and `openfisca-extension-template` are only needed for testing. If that is indeed the case, then the task should be renamed and the above comment corrected:
```suggestion
# Enable regression testing with template repositories
install-test-deps:
```